modules/glnexus/main.nf

```diff
@@ -0,0 +1,49 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process GLNEXUS {
+    tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::glnexus=1.4.1" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "docker://clinicalgenomics/glnexus:v1.4.1"
+    } else {
+        container "clinicalgenomics/glnexus:v1.4.1"
```

Comment:
```suggestion
        container "quay.io/biocontainers/glnexus:1.4.1--h40d77a6_0"
```

Reply:
Almost missed this 😅 If the tool is on Bioconda the containers shouild be available on Quay.io

Reply:
See [here](https://quay.io/repository/biocontainers/glnexus?tag=latest&tab=tags)

Reply:
aha! Did not know that :D Will fix this in the upcoming commit!